### PR TITLE
Alignment and usability improvements.

### PR DESCRIPTION
**Pull request by Jakar510**
• Title, Description, Hint, and ValueText Alignments
• Select all on Entry Cell Tap Binding
• Section Title, Header and Footers Text Alignments

### CODE REVIEW

**Comment by Jakar510** [comment]
https://github.com/Jakar510/AiForms.SettingsView/blob/TextAlignment/Changes.md

File: SettingsView.Droid/SettingsViewRecyclerAdapter.cs [hunk 1]
```diff
@@ -60,20 +60,26 @@ void _settingsView_ModelChanged(object sender, EventArgs e)
 
         void OnSectionPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
         {
+	        if ( !( sender is Section section ) ) return;
```
Comment (muak) [inline SettingsView.Droid/SettingsViewRecyclerAdapter.cs:63]: What purpose is this check?

Comment (Jakar510) [reply SettingsView.Droid/SettingsViewRecyclerAdapter.cs:63]: It reduces bugs by preventing cast exceptions. This is one of many areas that can be improved in this way.

**Comment by Jakar510** [comment]
May I make a new version of the library, and post to nuget? There are several things I'd like to do, expand, and improve. 

To name a few:

- Event Handlers for every cell, (Example: Check Value Changed for the CheckBoxCell)
- Grid-based layout instead of relative layout, for improved support for text alignment and multi-line, extending this pull request.
- Eliminate obscure exceptions by null checking and throwing exceptions as needed throughout; possibly even implement nullable aware contexts.
- Entry Cell HintText and ValueText overlaps on iOS.
- Move ValueText property to CellBase, even if it's not used on all cells (check and switch cells)
- improve consistency in the library
- Add a couple of other cell types.

**Comment by muak** [comment]
@Jakar510 
Those are great ideas.

You can freely develop the library on your fork repository.
Also, you can release a new package as long as you follow the MIT License.
However, if you do, change the package name from "AiForms.SettingsView" not to confuse, and don't include the icon.

If you want to continue doing the pull request,
Please read the [contribution guideline](https://github.com/muak/AiForms.SettingsView/blob/development/CONTRIBUTING.md) since I put it.

Thanks.

**Comment by Jakar510** [comment]
Thanks. If you want to accept the pull request, you can by all means incorporate it. I'll start working on the fork soon.

**Comment by muak** [comment]
@Jakar510 
I want to prioritize stabilizing the current version, so I may not be able to incorporate the changes right away.
Thank you for your understanding.

**Pull request closed without merging**